api-reference/10 UI Components/dxMap/1 Configuration/routes/mode.md

```diff
@@ -8,4 +8,28 @@ default: 'driving'
 Specifies a transportation mode to be used in the displayed route.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+Choose a predefined route mode, such as `'driving'` or `'walking'`, or select one supported by your provider:
```

Comment:
So these two modes that you mention - `driving` and `walking` - can I specify them regardless of the provider? 

The lists below show that Azure doesn't have a `walking` mode. It's `pedestrian`. Can I still specify `walking`? 

And if I understand the above correctly... What other "predefined route modes" are available? Why do you only specify a couple of examples? I think a complete list would be useful.

Reply:
There are only 2 predefined modes: 'walking' and 'driving'. You can specify them regardless of the provider (Map converts them to the necessary modes, e.g. you specify "walking" for Azure and it works).

Reply:
Then I would be specific that these two predefined modes are the only two predefined modes available. In other words, I wouldn't use the phrase "such as", as it may mean that there are additional predefined modes. 

```suggestion
Regardless of the current provider, you can specify one of two predefined modes: `'driving'` or `'walking'`. You can also select a provider-specific mode:
```